xmodule/seq_block.py

```diff
@@ -471,6 +471,11 @@ def public_view(self, context):
         return self._student_or_public_view(context or {}, prereq_met, prereq_meta_info, None, PUBLIC_VIEW)
 
     def author_view(self, context):  # lint-amnesty, pylint: disable=missing-function-docstring
+        print('@@@@@@@@@@@@@@@@@@@@@@@@@@@')
```

Comment:
Nit: remind myself to get rid of this

Reply:
Done in https://github.com/openedx/edx-platform/pull/33263